### PR TITLE
Add Time dimension for RU language

### PR DESCRIPTION
**Pull request by landyrev**
Used b40e2147a9e7e5445e3c42ffc4b45b30d3b1b052 as reference

### CODE REVIEW

File: Duckling/Time/RU/Rules.hs [hunk 1]
```diff
@@ -0,0 +1,1122 @@
...
+
+ruleInstants :: [Rule]
+ruleInstants = mkRuleInstants
+  [ ( "сейчас"       , TG.Second ,  0, "сейчас"               )
+  , ( "сегодня"      , TG.Day    ,  0, "сегодня"              )
+  , ( "завтра"       , TG.Day    ,  1, "завтра"               )
+  , ( "вчера"        , TG.Day    , -1, "вчера"                )
+  , ( "послезавтра"  , TG.Day    ,  2, "послезавтра"          )
+  , ( "позавчера"    , TG.Day    , -2, "позавчера"            )
+  , ( "Конец месяца" , TG.Month  ,  1, "(конец|когца) месяца" )
```
[the remark this answers is not in the document]
Comment (landyrev) [reply Duckling/Time/RU/Rules.hs]: thank you, fixed it

**Comment by landyrev** [comment]
Merge conflicts resolved

**Comment by chessai** [comment]
@landyrev @krizka this is a really useful commit, and i want to get it merged. However, tests indicate a number of correctness bugs. It looks like the rules for "week" are non-existent, and the rules for "last <time>" are also failing. There are many more tests failing, as well it seems. Would one of you mind taking over the PR and fixing these bugs/adding necessary rules to make them work?

File: Duckling/Time/RU/Rules.hs [hunk 2]
```diff
@@ -0,0 +1,1122 @@
...
+
+ruleDaysOfWeek :: [Rule]
+ruleDaysOfWeek = mkRuleDaysOfWeek
+  [ ( "Понедельник" , "понедельник(а)?|пн" )
+  , ( "Вторник"     , "вторник(а)?|вт"     )
+  , ( "Среда"       , "сред(а|у)|ср"       )
+  , ( "Четверг"     , "четверг(а)?|чт"     )
+  , ( "Пятница"     , "пятниц(а|у)|пт"     )
+  , ( "Суббота"     , "суббот(а|у)|сб"     )
+  , ( "Воскресенье" , "воскресенье|вс"     )
```
Comment (kappa) [inline Duckling/Time/RU/Rules.hs:52]: 3-letter abbreviations are also pretty popular, "пнд/втр/срд/чтв/птн/сбт|суб/вск".

File: Duckling/Time/RU/Corpus.hs [hunk 3]
```diff
@@ -0,0 +1,521 @@
...
+             [ "через 3 года"
+             ]
+  , examples (datetime (2013, 2, 19, 4, 0, 0) Hour)
+             [ "через 7 дней"
+             ]
+  , examples (datetime (2013, 2, 19, 0, 0, 0) Day)
+             [ "через 1 неделю"
+             ]
+  , examples (datetime (2013, 2, 5, 4, 0, 0) Hour)
+             [ "7 дней назад"
```
Comment (kappa) [inline Duckling/Time/RU/Corpus.hs:305]: \+ "7 дней тому назад"

**Review by kappa — COMMENTED** [review]
Thank you for working on this! This is awesome!

I added some in-line comments.

Other cases that I believe would improve parsing:
1. "позапозавчера", "послепослезавтра".
2. "20-го октября", "8-е марта".
3. "в октябре 20 числа", "март, 3-е число".
4. "в полночь", "2 часа пополудни".
5. "полшестого", "без четверти 11", "через пару минут".

**Comment by landyrev** [comment]
Thank you @chessai, @kappa !

Will get back to PR by the end of the week

**Comment by chessai** [comment]
closed by 590651150bd51d69dd9f73872b3a869e0842fd97

note that master is still missing some failures from the corpus/any improvements mentioned here. any bugs/incorrect bits mentioned in review were fixed.

i would say that if you wish to continue, just continue off of master.

**Pull request closed without merging**